scripts/events.py

```diff
@@ -1671,13 +1671,27 @@ def handle_injuries_or_general_death(self, cat):
             return True
 
         # chance to die of old age
-        age_chance = game.config["death_related"]["old_age_death_chance"]
         age_start = game.config["death_related"]["old_age_death_start"]
-        if cat.moons > int(
-                random.random() * age_chance) + age_start:  # cat.moons > 150 <--> 200
-            
-            Death_Events.handle_deaths(cat, other_cat, game.clan.war.get("at_war", False),
-                                            enemy_clan, alive_kits)
+        # map game_config values to the multipliers to make the config values more user-friendly
+        death_curve_setting = game.config["death_related"]["old_age_death_curve"]
+        death_curve_multipliers = {
+            1: 0.001,
+            2: 0.002,
+            3: 0.003,
+            4: 0.0045,
+            5: 0.008,
+            6: 0.02
+        }
+        death_curve_value = death_curve_multipliers[death_curve_setting]
```

Comment:
I think limiting values to only 6 possible settings is non-optimal. game.config can be used by players, but it's primary purpose to allow easy re-balancing without changing any code.  Limiting the death_curve_setting to only six settings means that any small adjustments need to be made in the code itself.   

This particular line also introduces a crash if death_curve_setting is not an integer 1-6.  That can be fixed by using `get()` with an appropriate default value, but I would suggest the storing the actual value of the multiplier in config.

Reply:
Good point! I have changed it to a more direct value now. It is multiplied by 0.001 in the actual formula, just to make it look neater, since the only viable ranges for the formula are between ~0.001 and ~0.02.

Reply:
Awesome, this looks great!